apps/api/nest-cli.json

```diff
@@ -20,11 +20,11 @@
         "outDir": "dist/src"
       },
       {
-        "include": "app/content-templates/usecases/compile-template/templates/layout.handlebars",
+        "include": "app/layouts/usecases/get-novu-layout/templates/layout.handlebars",
         "outDir": "dist/src"
       },
       {
-        "include": "app/content-templates/usecases/compile-template/templates/basic.handlebars",
+        "include": "app/content-templates/usecases/compile-email-template/templates/basic.handlebars",
         "outDir": "dist/src"
```

Comment:
Refactored the locaiton of those to live closely to the relevant parts where they belong to